### PR TITLE
feat: update styling

### PR DESCRIPTION
**Pull request by gracescharf**
## Description

* grabs the styling and layout changes from garbage-fire branch
* i'm noticing that the company overview isn't receiving the data when summary stats are -- even though they're both listening for tickerSelections$ which has the 'shareReplay(1)' on it  
## Related Issues

- Closes ISSUE

## Testing

1. Step 1

### CODE REVIEW

File: components/CompanyOverview/CompanyOverview.module.scss [hunk 1]
```diff
@@ -1,13 +1,38 @@
+@import "../../styles/variables.scss";
```
Comment (iprado78) [inline components/CompanyOverview/CompanyOverview.module.scss]: Use '@/styles/...'

File: components/CompanyOverview/CompanyOverview.module.scss [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+
+    h3 {
+      margin-top: 0.5rem;
+    }
+
+    h1 {
+      margin: 0;
+    }
+
+    h3 {
```
Comment (iprado78) [inline components/CompanyOverview/CompanyOverview.module.scss:26]: Duplicate h3 declaration.  Consolidate.  Also, I think semantically these should be h2

File: components/CompanyOverview/CompanyOverview.tsx [hunk 3]
```diff
@@ -28,11 +28,18 @@ export default function CompanyOverview() {
...
-        src={companyOverview.imgUrl}
-        alt={`Overview of ${companyOverview.companyName}`}
-      />
+      <header>
+        <div className={styles.left}>
+          <h1>{companyOverview.companyName}</h1>
+          <h3>CEO: {companyOverview.ceo} </h3>
+        </div>
+        <div className={styles.right}>
+          <img
```
Comment (iprado78) [inline components/CompanyOverview/CompanyOverview.tsx]: We should start using that Next image component.

File: components/Footer.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import styles from "@/styles/Home.module.css"
+import styles from "@/styles/Home.module.scss"
```
Comment (iprado78) [inline components/Footer.tsx:2]: May be overkill, but I think we should move the styles out of Home and into a Footer.module.scss. Ditto for the other Home components.

File: components/Header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  title,
+  children,
+}: {
+  title: string
+  children: ReactNode
+}) {
+  return (
+    <header className={styles.container}>
+      <div className={styles.left}>
+        <img src="USBankLogo.jpeg" alt="US Bank Logo" width="200" />
```
Comment (iprado78) [inline components/Header/Header.tsx:14]: See other comment re: Image component

File: components/Main.tsx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, { ReactNode } from "react"
-import styles from "@/styles/Home.module.css"
```
Comment (iprado78) [inline components/Main.tsx:2]: See above.  Header.module.scss

File: components/StatsTable/StatsTable.module.scss [hunk 7]
```diff
@@ -1,10 +1,11 @@
+@import "../../styles/variables.scss";
```
Comment (iprado78) [inline components/StatsTable/StatsTable.module.scss]: "@/styles/.."

File: components/SummaryStats/SummaryStats.tsx [hunk 8]
```diff
@@ -35,9 +35,6 @@ const [useSummaryStats] = bind(
 
 export default function SummaryStats() {
   const summaryStats = useSummaryStats()
-  if (!Object.keys(summaryStats).length) {
```
Comment (iprado78) [inline components/SummaryStats/SummaryStats.tsx:38]: Good catch that this is no longer necessary

File: components/Tabs/Tabs.tsx [hunk 9]
```diff
@@ -73,7 +73,12 @@ export default function Tabs({ tabLabels, children }: ITabsProps) {
   const hasTickerSelection = useHasTickerSelection()
 
   if (!hasTickerSelection) {
-    return <div>Nothing to see here</div>
+    return (
+      <div>
+        <p>Nothing to see here...</p>
+        <img src="https://media.giphy.com/media/BEob5qwFkSJ7G/giphy.gif" />
```
Comment (iprado78) [inline components/Tabs/Tabs.tsx]: Nice.  Image component.  Also, remember alt attribute!

**Review by iprado78 — CHANGES_REQUESTED** [review]

File: styles/variables.scss [hunk 10]
```diff
@@ -0,0 +1,5 @@
+// breakpoints
+$tablet: 768px;
```
Comment (iprado78) [inline styles/variables.scss:2]: 💯

File: components/Footer.tsx [hunk 11]
```diff
@@ -1,5 +1,5 @@
...
 
 export default function Footer() {
   return (
```
Comment (iprado78) [inline components/Footer.tsx:5]: Also, this Footer is pretty broken on the mobile view.  Maybe we just kill it until we have time to make a true sticky but responsive footer.

File: components/Main.tsx (hunk 6 above)
Comment (gracescharf) [reply components/Main.tsx:2]: noted -- might make this a todo tho?

File: components/Tabs/Tabs.tsx [hunk 12]
```diff
@@ -73,7 +74,20 @@ export default function Tabs({ tabLabels, children }: ITabsProps) {
...
 
   if (!hasTickerSelection) {
-    return <div>Nothing to see here</div>
+    return (
+      <div>
+        <p>Nothing to see here...</p>
+        <div>
+          <Image 
+            src="https://media.giphy.com/media/BEob5qwFkSJ7G/giphy.gif"
+            alt="Spongebob looking very sad with teary eyes and a quivering mouth."
```
Comment (iprado78) [inline components/Tabs/Tabs.tsx:83]: 🦖

File: styles/z-order.scss [hunk 13]
```diff
@@ -0,0 +1,17 @@
+/*
```
Comment (iprado78) [inline styles/z-order.scss:1]: 🥳

**Review by iprado78 — DISMISSED** [review]

**Review by iprado78 — APPROVED** [review]

**Pull request merged**